### ISSUE TITLE
Progression of CombineWorker

### ISSUE THREAD

**Issue by virtualzeta** [Maksimka101/combine#24]
Is there a way to get some progress data from isolates using CombineWorker()?

**Comment by Maksimka101** [comment]
Hello

For now, there is no way to send progress data from worker. You can use the `CombineIsolate` for this

**Comment by virtualzeta** [comment]
I supposed.
And is there a way to use CombineIsolate to wait, get and manage a result like with CombineWorker, so as to then also have the possibility to send other messages (such as progression data)?

**Comment by Maksimka101** [comment]
No. There is no such methods as 'CombineWorker.execute'. You have to send an event, handle it in isolate, send back result and handle it
